handler.fish

```diff
@@ -2,7 +2,7 @@ function __fish_command_not_found_on_interactive --on-event fish_prompt
     functions --erase __fish_command_not_found_handler
     functions --erase __fish_command_not_found_setup
```

Comment:
Do we also need this, now?

```suggestion
    functions --erase __fish_command_not_found_handler
    functions --erase __fish_command_not_found_setup
    functions --erase fish_command_not_found
```

Reply:
I’m not a Fish user so I can’t help you here. Maybe @xu-cheng?

Reply:
Yeah, I'm not either 😬

It does seem to work fine for me using Homebrew's fish (3.2.2) locally both with and without this line. That's all I can say though...

Reply:
FYI, I am not a fish user as well. I implemented this because some users asked for it back the day. 

As long as it works locally, I think it should be fine.

Reply:
Thanks!

@bfontaine: okay if I merge this, then?

Reply:
Yes 👍